subgraphs/apeswap/src/utils/constant.ts

```diff
@@ -0,0 +1,152 @@
+import { BigDecimal, BigInt, Address, dataSource } from "@graphprotocol/graph-ts";
+import { Factory } from "../../generated/Factory/Factory";
+
+////////////////////////
+///// Schema Enums /////
+////////////////////////
+
+export namespace Network {
+  export const AVALANCHE = "AVALANCHE";
+  export const AURORA = "AURORA";
+  export const BSC = "BSC";
+  export const CELO = "CELO";
+  export const CRONOS = "CRONOS";
+  export const ETHEREUM = "ETHEREUM";
+  export const FANTOM = "FANTOM";
+  export const HARMONY = "HARMONY";
+  export const MOONBEAM = "MOONBEAM";
+  export const MOONRIVER = "MOONRIVER";
+  export const OPTIMISM = "OPTIMISM";
+  export const POLYGON = "POLYGON";
+  export const XDAI = "XDAI";
+}
+
+export namespace ProtocolType {
+  export const EXCHANGE = "EXCHANGE";
+  export const LENDING = "LENDING";
+  export const YIELD = "YIELD";
+  export const BRIDGE = "BRIDGE";
+  export const GENERIC = "GENERIC";
+}
+
+export namespace RewardTokenType {
+  export const DEPOSIT = "DEPOSIT";
+  export const BORROW = "BORROW";
+}
+
+export namespace LiquidityPoolFeeType {
+  export const FIXED_TRADING_FEE = "FIXED_TRADING_FEE";
+  export const TIERED_TRADING_FEE = "TIERED_TRADING_FEE";
+  export const DYNAMIC_TRADING_FEE = "DYNAMIC_TRADING_FEE";
+  export const FIXED_PROTOCOL_FEE = "FIXED_PROTOCOL_FEE";
+  export const DYNAMIC_PROTOCOL_FEE = "DYNAMIC_PROTOCOL_FEE";
+}
+
+export namespace TransferType {
+  export const MINT = 'MINT'
+  export const BURN = 'BURN'
+  // Pool addresses are also stored in the HelperStore
+}
+
+export const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+// export const FACTORY_ADDRESS = "0x0841BD0B734E4F5853f0dD8d7Ea041c241fb0Da6";
+export const FACTORY_ADDRESS = dataSource.network() == Network.BSC.toLowerCase() ? BSC.FACTORY_ADDRESS : POLYGON.FACTORY_ADDRESS;
+export let factoryContract = Factory.bind(Address.fromString(FACTORY_ADDRESS));
+
+export const DEFAULT_DECIMALS: i32 = 18;
+export const USDC_DECIMALS: i32 = 6;
+export const USD_DENOMINATOR = BigInt.fromI32(10 ** 18).toBigDecimal();
+export const FEE_DENOMINATOR = BigInt.fromI32(10 ** 10);
+export const FEE_DECIMALS = 10;
+export const TRADING_FEE = toPercentage(BigDecimal.fromString("0.2"));
```

Comment:
Put these in config

Reply:
Please do not resolve a comment that requires further discussion

Reply:
what you can do is to factor all bsc/polygon-specific code into a separate file or a separate folder

Reply:
Fixed. So I move all bsc/polygon specific code to file "networkConfig"